### PR TITLE
Describe HtmlEditor - OutputFormats Demo

### PR DESCRIPTION
**Pull request by czernous-devexpress**

### CODE REVIEW

File: JSDemos/Demos/HtmlEditor/OutputFormats/jQuery/description.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Our HtmlEditor outputs the markup in HTML and Markdown formats. To specify the format, use the [valueType] property. If you use Markdown, link the <a href="https://www.npmjs.com/package/turndown" target="_blank" rel="noopener">turndown</a> and <a href="https://www.npmjs.com/package/showdown" target="_blank" rel="noopener">showdown</a> libraries before the DevExtreme scripts.
+
+The HtmlEditor stores the markup in the [value] property. In this demo, the value is displayed under the HtmlEditor. You can use the Html and Markdown buttons to switch between output formats.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/HtmlEditor/OutputFormats/jQuery/description.md]: Links?

**Review by RomanTsukanov — DISMISSED** [review]

File: JSDemos/Demos/HtmlEditor/OutputFormats/description.md [hunk 2]
```diff
@@ -1 +1,3 @@
-In&nbsp;this demo, you can switch between the HTML and Markdown output formats and view the result below the HtmlEditor.
\ No newline at end of file
+Our HtmlEditor outputs the markup in HTML and Markdown formats. To specify the format, use the [valueType](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#valueType) property. If you use Markdown, import the Markdown converter as described in the **valueType** article.
+
+The HtmlEditor stores the markup in the [value](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#value) property. In this demo, the value is displayed under the HtmlEditor. You can use the Html and Markdown buttons to switch between output formats.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/HtmlEditor/OutputFormats/description.md]:
```suggestion
The HtmlEditor stores the markup in the [value](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#value) property. In this demo, the value is displayed under the HtmlEditor. You can click the Html and Markdown buttons to switch between output formats.
```

File: JSDemos/Demos/HtmlEditor/OutputFormats/jQuery/description.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Our HtmlEditor outputs the markup in HTML and Markdown formats. To specify the format, use the [valueType](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#valueType) property. If you use Markdown, link the <a href="https://www.npmjs.com/package/turndown" target="_blank" rel="noopener">turndown</a> and <a href="https://www.npmjs.com/package/showdown" target="_blank" rel="noopener">showdown</a> libraries before the DevExtreme scripts.
+
+The HtmlEditor stores the markup in the [value](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#value) property. In this demo, the value is displayed under the HtmlEditor. You can use the Html and Markdown buttons to switch between output formats.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/HtmlEditor/OutputFormats/jQuery/description.md]:
```suggestion
The HtmlEditor stores the markup in the [value](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#value) property. In this demo, the value is displayed under the HtmlEditor. You can click the Html and Markdown buttons to switch between output formats.
```

**Review by dmitry-eliseev-devexpress — DISMISSED** [review]
A minor change.

File: JSDemos/Demos/HtmlEditor/OutputFormats/description.md [hunk 4]
```diff
@@ -1 +1,3 @@
-In&nbsp;this demo, you can switch between the HTML and Markdown output formats and view the result below the HtmlEditor.
\ No newline at end of file
+Our HtmlEditor outputs the markup in HTML and Markdown formats. To specify the format, use the [valueType](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#valueType) property. If you use Markdown, import the Markdown converter as described in the **valueType** article.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/HtmlEditor/OutputFormats/description.md]:
```suggestion
Our HtmlEditor can output the markup in HTML and Markdown formats. To specify the format, use the [valueType](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/#valueType) property. If you use Markdown, import the Markdown converter as described in the **valueType** article.
```

**Review by dirk-pieterse — DISMISSED** [review]

**Review by AlexJameson — APPROVED** [review]

**Pull request merged**